platform/darwin/docs/guides/For Style Authors.md.ejs

```diff
@@ -178,21 +178,22 @@ source object is a member of one of the following subclasses of `MGLSource`:
 
 In style JSON | In the SDK
 --------------|-----------
+`vector`      | `MGLVectorTileSource`
+`raster`      | `MGLRasterTileSource`
+`raster-dem`  | `MGLRasterDEMSource`
```

Comment:
Should this be MGLRasterDemTileSource in case we decide to add a computed RasterDEM source in conjunction with computed raster/image source?

Reply:
Do you think that’s at least as likely as a computed image source? If so, I think I’d push for some `mbgl::style::Source` classes to be consolidated, which would allow us to consolidate these classes at the SDK level.

Reply:
I haven't seen any requests for it. And you make a good point, core sources will probably get refactored if we get that far.